xml/installation-installation-gui_installer.xml

```diff
@@ -159,21 +159,20 @@
     </para>
     <substeps>
      <step>
-      <para>
-       Update the zypper configuration file.
-      </para>
-  <screen>
-<?dbsuse-fo font-size="0.65em"?>
-sudo sed -i -e "s,.*solver.allowVendorChange.*,solver.allowVendorChange = true," /etc/zypp/zypp.conf
-  </screen>
-     </step>
-     <step>
-      <para>
-       Mount the repo as <literal>DC8S</literal>:
+      <para vendor="suse">
+       Mount the repository as <literal>&cloud;</literal>:
       </para>
-      <!-- FIXME: This probably needs to be profiled for HPE since the HPE
-      ISO has a different name. - sknorr, 2018-02-28 -->
-      <screen>sudo zypper ar -G "iso:///?iso=$(ls ~/stack/SUSE-OPENST*.iso)" DC8S</screen>
+      <screen vendor="suse">
```

Comment:
The following screens should not be indented. And you don't need to the font-size PI, these are less than 80 characters.